src/app/components/Article.stories.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Why is the ignore linting required?

Reply:
@pjlee11 because it thinks it should be a dep not a dev-dep and it's wrong.